liqd_product/assets/scss/_line_awesome_overwrites.scss

```diff
@@ -55,3 +55,8 @@
     font-family: 'FontAwesome', sans-serif;
     content: '\F266'; // lightbulb
 }
+
+.fa-comment:before {
+    font-family: 'FontAwesome', sans-serif;
+    font-style: normal;
```

Comment:
Don't you need the content anymore?

Reply:
No, it already comes from somewhere :/ Not sure, philli probably knows why

Reply:
content wasn't changing icon, was set with classname, but i've just seen now it was meant to be a lightbulb not a comment, which i didnt know... maybe we need to find updated lightbulb content code?

Reply:
ahh thats why it was in the overwrites, god im slow today, sorry! is it meant to be a lightbulb?

Reply:
Oups, guess I shouldn't have pushed it already :)

Reply:
Yeah I don't get why its a lightbulb? old designs are archived on zeplin so I can't access them, we can ask Sandy on Monday

Reply:
Haha, it's all fine! I think it's meant to be a comment and if it's back now without the content (which I don't get), we're fine!